### DISCUSSION TITLE
Demande info : durée max de réservation

### DISCUSSION THREAD

**Discussion by eleo56** [JeromeDevome/GRR#346] [Q&A]
Bonjour,

Merci pour ce logiciel qui semble correspondre à notre besoin. Afin d'être sûr de ne pas l'installer pour rien, je souhaite obtenir une confirmation.

Est-ce possible de fixer une durée de réservation maximale pour un domaine (à priori ce serait 2h ou 3H)?

Je crois avoir trouvé la réponse (champ « Durée maximale en minutes pour une réservation » vu dans ce tuto https://blogacabdx.ac-bordeaux.fr/resdane/wp-content/uploads/sites/9/2015/01/Arogs2_GRR_doc_complete.pdf ) mais je souhaite être sûr que l’on ait la main et que ce ne soit pas un choix entre : 1 journée / pas de limitation.
C’est un critère important pour nous. 

Je vous remercie et vous souhaite une bonne journée
Eléonore

**Comment by ynaessens** [comment]
Bonjour,
oui, ce paramètre est défini pour une réservation, et par domaine. La durée de 1440 minutes est donnée à titre indicatif.
Pour deux heures, vous définirez la durée à 120.
Au niveau ressource, il existe également des paramètres gérant la quantité et les délais de réservation.
Cordialement,
YN

**Reply by eleo56** [reply]
Merci!
